public/locales/fi/common.json

```
@@ -40,7 +40,8 @@
     "micro": "Lataa Testaustime Micro:lle",
     "neovim": "Lataa Testaustime Neovim:lle",
     "title": "Laajennukset",
-    "vscode": "Lataa Testaustime Visual Studio Code:lle"
+    "vscode": "Lataa Testaustime Visual Studio Code:lle",
+    "sublime": "Lataa Testaustime Sublime Text:ille"
```

I'd change it to "Text:lle" to match the others. I don't think "ille" is a valid suffix in Finnish if it comes after a colon (though don't quote me on that :smile:)
```suggestion
    "sublime": "Lataa Testaustime Sublime Text:lle"
```